GOVERNANCE.md

```diff
@@ -2,53 +2,54 @@
+* Defining the governance structure of the CNB project
+* Creating new teams, reorganizing teams, and removing existing teams
+* Delegating responsibilities to teams and re-allocating responsibilities amongst teams
+* Appointing Team Leads
+* Defining the [RFC process](https://github.com/buildpacks/rfcs#rfc-process) through which cross-cutting changes are proposed and approved
 * Defining the annual roadmap
 * Anything else that falls through the cracks
 
+New TOC members must be nominated by an existing TOC member and elected by a super-majority of the TOC. TOC members may graduate to emeritus status by request or, in exceptional circumstances, by a super-majority vote of the TOC.
+
 ## Teams
 CNB Teams are responsible for narrower sets of concerns related to specific aspects of the project.
 
 ### Roles
 
 #### Team Leads
 
-Each team has a Team Lead. The Team Lead is a maintainer who is appointed by the TOC to be responsible for:
+Each team has a Team Lead. The Team Lead is a maintainer who has special responsibilites for representing team concerns at the project level.
 
-* Casting binding votes on RFCs owned by their team
-* Moving an RFC to a status of "voting"
-* Assigning an end-date for voting on their team’s RFCs
-* Delegating another maintainer from the same Team to temporarily fulfill these responsibilities
+These additional responsibilities include:
+* Casting binding votes on RFCs as described in the RFC process.
+* Stewarding project RFCs owned by the team as described in the RFC process.
+* Representing the work of the team to the TOC.
+* Delegating another maintainer from the same Team to temporarily fulfill these responsibilities.
 
 #### Maintainers
-Maintainers are in charge of the day to day maintenance of the team's projects. They review, approve, and merge PRs, ensuring contributions align with project goals and meet the project's quality standards.
+Maintainers are in charge of the day to day maintenance of the team's projects including:
+* ensuring contributions align with project goals and meet the project's quality standards. 
+* reviewing, approving, and merging PRs.
+* planning release milestones, and releasing components under the team's area of responsibility.
+* representing the work of the team to the community.
+* supporting contributors.
+* growing the team by mentoring aspiring contributors and maintainers.
 
-New maintainers must already be contributors, must be nominated by an existing maintainer, and must be elected by a [supermajority](#supermajority) of CNB Team Leads. Likewise, maintainers can be removed by a supermajority of Team Leads or can resign by notifying one of the maintainers.
+New maintainers must already be contributors, must be nominated by an existing maintainer, and must be elected by a [supermajority](#supermajority) of CNB Team Leads and the TOC. Likewise, maintainers can be removed by a supermajority of Team Leads or can resign by notifying one of the maintainers.
```

Comment:
```suggestion
New maintainers must already be contributors, must be nominated by an existing maintainer, and must be elected by a [supermajority](#supermajority) of CNB Team Leads and the TOC. Likewise, maintainers can be removed by a supermajority of Team Leads and the TOC or can resign by notifying one of the maintainers.
```

Reply:
resolved in https://github.com/buildpacks/community/pull/166/commits/26201493c1b56dd97faad3eb3bbf977d793df7d2